pkg/gadgets/params.go

```diff
@@ -28,12 +28,13 @@ const (
 )
 
 const (
-	K8SNodeName      params.ValueHint = "k8s:node"
-	K8SNodeList      params.ValueHint = "k8s:node-list"
-	K8SPodName       params.ValueHint = "k8s:pod"
-	K8SNamespace     params.ValueHint = "k8s:namespace"
-	K8SContainerName params.ValueHint = "k8s:container"
-	K8SLabels        params.ValueHint = "k8s:labels"
+	LocalContainerName params.ValueHint = "local:containername"
```

Comment:
Since we also have `K8SContainerName` => `k8s:container` etc., we should either add `name` like `k8s:...name` to those fields or use `local:container` here. I don't have a strong preference, but we should keep it consistent (especially because later on the strings will be used by third parties.)

Reply:
Updated.